internal/dag/gatewayapi_processor.go

```diff
@@ -81,8 +81,7 @@ func (p *GatewayAPIProcessor) Run(dag *DAG, source *KubernetesCache) {
 	}
 
 	if len(p.source.gateway.Spec.Addresses) > 0 {
-		p.Error("Spec.Addresses is unsupported")
-		errs = append(errs, field.NotSupported(path, p.source.gateway.Spec.Addresses, []string{}))
+		errs = append(errs, &field.Error{Type: field.ErrorTypeNotSupported, Field: path.String(), BadValue: p.source.gateway.Spec.Addresses, Detail: "Spec.Addresses is not supported"})
```

Comment:
nit: maybe `errs` could be renamed to `gatewayErrs` or something, just so it is a little clearer what it is used for?

Reply:
👍